File: 2017-08-08/StringCalculator.cs
public class StringCalculator
{
   public int Add( string input )
   {
      if ( input == "" )
      {
         return 0;
      }
      int result = 0;
      string convertedInput = input.Replace( '\n', ',' );
      if ( convertedInput.Contains( "," ) )
      {
         string[] separatedInput = convertedInput.Split( ',' );
         for ( int i=0; i<separatedInput.Length; i++ )
         {
            result += System.Convert.ToInt32( separatedInput[i] );
         }
         return result;
      }
      else
      {
         return System.Convert.ToInt32( convertedInput );
      }
   }
}

File: 2017-09-04/python/StringCalculator.py
class StringCalculator:

   #def __init__(self):

   def add( self, str ):
      if ( str == '' ):
         return 0
      if ( str.find(',') ):
         values = str.split(',')
         result = 0
         for x in values:
            result += int(x)
         return result
      return int( str )

